cumulus_library_data_metrics/q_date_in_lifetime/q_date_in_lifetime.py

```diff
@@ -0,0 +1,25 @@
+"""Module for generating q_ref_target_pop tables"""
+
+import cumulus_library
+import os
+
+from cumulus_library_data_metrics.base import MetricMixin
+from cumulus_library_data_metrics.resource_info import DATES
+
+
+class TargetPopBuilder(MetricMixin, cumulus_library.BaseTableBuilder):
+    name = "q_date_in_lifetime"
+
+    def make_table(self, **kwargs) -> str:
+        self.add_summary_entry(kwargs["src"], self.render_sql("summary", **kwargs))
```

Comment:
I think you're passing the denominator sql in as the `group` field - I think you want `denominator=self.render_sql()` - and maybe rename summary.sql to denominator.sql for clarity.